cocktails9/cocktails9/Views/login.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+
+struct login: View {
```

Comment:
Naming convention of classes and structs in iOS should be CamelCase.
```suggestion
struct LoginView: View {
```

You can also apply this convention to file